apps/src/templates/Notification.jsx

```diff
@@ -184,6 +185,15 @@ const Notification = React.createClass({
               onClick={this.onAnnouncementClick}
             />
           )}
+          {buttonText && onButtonClick && (
+            <Button
+              color={Button.ButtonColor.gray}
+              text={buttonText}
+              style={styles.button}
+              target={newWindow ? "_blank" : null}
+              onClick={onButtonClick}
+            />
+          )}
```

Comment:
What if instead of adding a second/alternate `<Button/>` component, we just had `onAnnouncementClick` call `this.props.onButtonClick` if it exists?

Reply:
Thanks for the suggestion! That'll make the render much easier to read. I've pushed the changes you suggested. PTAL.